imcsdk/apis/server/bios.py

```diff
@@ -318,11 +337,11 @@ def set_boot_order_policy(handle, reboot_on_update=False,
         reboot_on_update (bool): True, False
         secure_boot (bool): secure boot
         boot_devices (list of dict): format
-            [{"order":'1', "type":"vmedia", "name":"vmedia"},
-             {"order":'2', "type":"lan", "name":"lan"}]
+            [{"order":'1', "device-type":"vmedia", "name":"vmedia"},
```

Comment:
is `vmedia` a valid type in `boot_order_policy` anymore? should this be `cdrom`/`fdd`?